### ISSUE TITLE
Rename repository

### ISSUE THREAD

**Issue by rmarkello** [netneurolab/pypyls#27]
Unfortunately, the current name of the repository, `pyls`, was [reserved on PyPi](https://pypi.org/project/pyls) as of nearly a decade ago. If this package is to ever be uploaded to PyPi for easy distribution (i.e., `pip install packagename`), it will need to be renamed to something that hasn't been used before.

It would be great if the name could somehow play on multivariate covariance models, decompositions, or dimensionality reduction&mdash;though writing those out sound a bit unwelcoming... All suggestions are welcome! 

**tl;dr**: Help! We need a new name!

**Comment by rmarkello** [comment]
Some (rather meager) options include:

`decompy`: Data **decom**positions in **Py**thon
`pyxcov`: **Py**thon-based **cross**-**cov**ariance analyses

**Comment by jacquelineharris** [comment]
pypls 🤷‍♀️
Hit a naming conflict when I installed so this would be helpful!

**Comment by liuzhenqi77** [comment]
Gonna have to work on this -- pyls is causing some conflict with python language server.
